python/cuml/fil/fil.pyx

```diff
@@ -39,8 +39,79 @@ from cuml.utils import get_cudf_column_ptr, get_dev_array_ptr, \
 cimport cuml.common.handle
 cimport cuml.common.cuda
 
+# from treelite cimport TreeliteModel, ModelHandle
+
 cdef extern from "treelite/c_api.h":
     ctypedef void* ModelHandle
+    cdef int TreeliteLoadXGBoostModel(const char* filename,
+                                      ModelHandle* out)
+    cdef int TreeliteLoadXGBoostModelFromMemoryBuffer(const void* buf,
+                                                      size_t len,
+                                                      ModelHandle* out);
+    cdef int TreeliteFreeModel(ModelHandle handle);
+    cdef int TreeliteQueryNumTree(ModelHandle handle, size_t* out)
+    cdef int TreeliteQueryNumFeature(ModelHandle handle, size_t* out)
+    # XXX untested
+    cdef int TreeliteLoadLightGBMModel(const char* filename,
+                                           ModelHandle* out)
+    # XXX untested
+    cdef int TreeliteLoadProtobufModel(const char* filename,
+                                           ModelHandle* out)
+
+cdef class TreeliteModel():
```

Comment:
Is it possible to put it in a different Python module (e.g. `cuml.treelite`) and simplify the function names and the class name (to just `Model`)? The method that loads everything into FIL from a file can stay in FIL, but with the name simplified to `load`.